frontend/util/federalistApi.js

```diff
@@ -5,15 +5,15 @@ import alertActions from '../actions/alertActions';
 
 export const API = '/v0';
 
-const csrfToken = typeof window !== 'undefined' ?
-  window.CSRF_TOKEN : global.CSRF_TOKEN;
-
-const defaultHeaders = {
-  'x-csrf-token': csrfToken,
-};
-
 export default {
   fetch(endpoint, params = {}) {
+    const csrfToken = typeof window !== 'undefined' ?
```

Comment:
Was this move to get the tests to behave after you require `federalistAPI`? Frustrating, but 🤷‍♀️, I don't have a better way to manage global state.

Reply:
yep, exactly